app/service/update-appeal-service.ts

```diff
@@ -374,34 +344,25 @@ export default class UpdateAppealService {
     }
 
     caseData.timeExtensions = [];
-    const previousAskForMoreTimes = appeal.previousAskForMoreTime;
 
-    if (previousAskForMoreTimes) {
-      previousAskForMoreTimes.forEach(askForMoreTime => {
-        this.addCcdTimeExtension(askForMoreTime, appeal, caseData);
-      });
-    }
```

Comment:
If we remove this we loose all time previous time extensions when we add a new one